app/src/main/kotlin/com/wire/android/ui/common/imagepreview/BulletHoleImagePreview.kt

```diff
@@ -41,13 +41,14 @@ fun BulletHoleImagePreview(imageUri: Uri, contentDescription: String) {
                     bottom.linkTo(parent.bottom)
                 }
         ) {
-            Image(
-                painter = rememberAsyncImagePainter(
-                    model = imageUri
-                ),
-                contentScale = ContentScale.Crop,
+            AsyncImage(
+                model = imageUri,
+                contentScale = ContentScale.FillWidth,
                 contentDescription = contentDescription,
-                modifier = Modifier.fillMaxSize()
+                modifier = Modifier.fillMaxSize(),
+                onSuccess = { it.result.drawable.apply { this.invalidateSelf() } },
+                onError = { it.result.drawable?.invalidateSelf() }, // handle error when not possible to load preview
+                error = rememberAsyncImagePainter(model = imageUri) // retry, fallback to load image when error
```

Comment:
Yes, indeed 👀 . I was having some attempts to reload the image on failure, since the coil component wasn't behaving well loading, sometimes a blank screen.

Anyway, I removed the `onError` and `onSuccess` blocks and maintained the `error` with a local fallback. We'll have to handle this in a future PR. 

Now the preview loader it's working better with no errors, using a Bitmap instead of a ByteArray 😖